inlong-dashboard/src/metas/sources/defaults/index.ts

```diff
@@ -26,18 +26,18 @@ export const allDefaultSources: MetaExportWithBackendList = [
     LoadEntity: () => import('../common/SourceInfo').then(r => ({ default: r.SourceInfo })),
   },
   {
-    label: 'Auto Push',
+    label: 'File',
```

Comment:
It is recommended to use the first alphabetical sorting for this sorting. As the subsequent types increase, the sorted order is more reasonable.

Reply:
Good point. I change it now.